lib/DueDatePicker/styles.scss

```diff
@@ -0,0 +1,349 @@
+}
+
+.duedate__submit--cancel {
+  font-size: $typo-size-slight;
+  margin-right: $layout-spacing-base / 2;
+  color: $neutral-dark;
+
+  &:hover {
+    text-decoration: none;
+    color: $neutral-dark;
+  }
+}
+
+.duedate__submit--save {
+  font-size: $typo-size-slight;
+  padding: $layout-spacing-base / 4 $layout-spacing-base / 2.5;
+}
+
+/* DueDate Calendar */
+
+.duedate__popover .DayPicker {
+  padding: $layout-spacing-base / 2;
+
+  &:after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+
+  .DayPicker-wrapper {
+    outline: 0;
+  }
+
+  .DayPicker-Caption {
+    text-align: center;
+    font-size: $typo-size-lead;
+    font-weight: $typo-weight-semi-bold;
+    padding: $layout-spacing-base / 4 0 $layout-spacing-base / 2;
+  }
+
+  .DayPicker-NavBar {
+    position: relative;
+  }
+
+  .DayPicker-NavButton {
+    position: absolute;
+    outline: 0;
+    cursor: pointer;
+    top: $layout-spacing-base / 2;
+    width: 12px;
+    height: 12px;
+  }
+
+  .DayPicker-NavButton:before {
+    border-style: solid;
+    border-width: 2px 2px 0 0;
+    border-color: $neutral-base;
+    content: '';
+    display: inline-block;
+    position: relative;
+    top: 2px;
+    transform: rotate(-45deg);
+    vertical-align: top;
+    height: $layout-spacing-base / 2.5;
+    width: $layout-spacing-base / 2.5;
+  }
+
+  .DayPicker-NavButton--next {
+    right: $layout-spacing-base / 4;
+
+    &:before {
+      left: 0;
+      transform: rotate(45deg);
+    }
+  }
+
+  .DayPicker-NavButton--prev {
+    left: $layout-spacing-base / 4;
+
+    &:before {
+      left: 4px;
+      transform: rotate(-135deg);
+    }
+  }
+
+  .DayPicker-Month {
+    margin: 0;
+  }
+
+  .DayPicker-Weekday, .DayPicker-Day {
+    text-align: center;
+    width: 28px;
+    height: 28px;
+    display: inline-block;
+    float: left;
+    overflow: hidden;
+    line-height: 28px;
+    margin: 0 6px;
+    font-size: $typo-size-slight;
+  }
+
+  .DayPicker-Weekday {
+    margin: 0;
+    width: $layout-spacing-base * 2;
+    color: $neutral-dark;
+  }
+
+  abbr {
+    text-decoration: none;
+    border-bottom: none;
+  }
+
+  .DayPicker-Day {
+    border: 1px solid transparent;
+    border-radius: 100%;
+    outline: none;
+
+    &.DayPicker-Day--today {
+      background-color: $neutral-lightest;
+    }
+
+    &.DayPicker-Day--selected {
+      background-color: $color-link-base;
+      color: white;
+
+      &:hover {
+        color: white;
+      }
+    }
+  }
+
+  .DayPicker-Day--outside {
+    color: $neutral-base;
+  }
+
+  .DayPicker-Day:hover {
+    border-color: $color-link-base;
+    color: $color-link-base;
+    cursor: pointer;
+  }
+
+  .DayPicker-Day--past {
+    &:hover {
+      border-color: $primary-red;
+      color: $primary-red;
+    }
+
+    &.DayPicker-Day--selected {
+      background-color: $primary-red;
+      color: white;
+    }
+  }
+}
+
+/* DueDate Time picker dropdown */
+
+.duedate__popover .duedate__header--time-picker {
```

Comment:
Is `duedate__header--time-picker` a modifier or a element? Reads like a element so maybe `duedate__time-picker`? Reducing the length of BEM selectors is always a win 😄